src/docs/guides/create-blazor-cms/README.md

```diff
@@ -730,6 +730,51 @@ Run the application. Click on the `Orchard Core` link on the Left Nav will load
 # Add multitenancy to the Blazor App
 One of the key features of Orchard Core is its multi-tenancy support. In this section, we will add multi-tenancy support to our Blazor application.
 
+> !!! NOTE
+> 
+> Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well can pose a problem.
+> A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```

Comment:
```suggestion
!!! note
    Orchard Core listens for requests, also on '/' (e.g. when a site needs to be configured). So having our Blazor app listening to '/' as well can pose a problem.
    A solution is to replace `@page "/"` with `@page "/home"` in our `home.razor` file (and update our `navmenu.razor`), so Blazor (the app) will listen to `/home` while Orchard Core is not hindered by that.
```

Reply:
Yes, and it is a critical issue for oc and blazor to 'work' together ... 
I did it right in the samples repo, but forgot to mention it in the docs. (My bad).